web/src/admin/credentials/IssueCredential.vue

```diff
@@ -5,6 +5,8 @@
 
     <section v-if="issuedCredential">
       <header>Issued Credential</header>
+      <button class="btn btn-primary" @click="copyToClipboard">Copy to Clipboard</button>
+      <br><br>
```

Comment:
better use some CSS for this (instead of `<br>`).

You can add a `<style>` tag just above the `<script>` tag if you want CSS only applicable to this component